pkg/controller/cluster_resources.go

```diff
@@ -169,3 +170,160 @@ func NodePoolFromResources(name string, pr *edgeproto.NodePoolResources) *edgepr
 		NodeResources: &nr,
 	}
 }
+
+// calcResourceScore gets a score which represents the available resources
+// in a cluster. A higher score means more available resources.
+func (s *ClusterInstApi) calcResourceScore(free resspec.ResValMap) uint64 {
```

Comment:
Eh, logic is so close between calculation from free and used...Maybe call them `calcResourceScoreFromFree` and `calcResourceScoreFromUsed` to mark the difference?

Reply:
I am talking about logic in `(s *CloudletResCalc) calcResourceScore()`

Reply:
Oh I see, one takes freeRes args and the other used. I'll rename the CloudletResCalc one to calcResourceScoreFromUsed, as in general resource scores are based on free resources.